components/oss/aliyun/oss.go

```diff
@@ -44,29 +44,24 @@ func NewAliyunOss() l8oss.Oss {
 }
 
 func (a *AliyunOSS) Init(ctx context.Context, config *l8oss.Config) error {
-	var connectTimeout, readWriteTimeout int64
+	connectTimeout, readWriteTimeout := 30, 60
```

Comment:
change the variables name to defaultConnectTimeoutSec dedaultReadWriteTimeoutSec?